packages/metrics/src/config/routes.ts

```diff
@@ -82,13 +83,30 @@ export enum EventType {
   MARRIAGE = 'marriage'
 }
 
+/*
+ * This route wrapper triggers a view update for materialised views
+ * we read data to Metabase from. If you are not seeing your data updated
+ * after a user action, you most likely need to add this wrapper to some
+ * new endpoint handler here.
+ */
+function analyticsDataRefreshingRoute<T extends Array<any>, U>(
+  handler: (...args: T) => U
+) {
+  // Do not use await for the refresh operation. This operation can take minutes or more.
+  // Consider triggering this a task that will be left to be run in the background.
```

Comment:
@rikukissa if there are outstanding improvements to be made here, can we track the technical debt in technical-improvements channel in Slack please?